spec/do_username_spec.rb

```diff
@@ -0,0 +1,87 @@
+RSpec.describe DOUsername do
+  describe '.generate' do
+    subject { described_class }
+
+    it 'responds with different usernames based on srand' do
+      [
+        'LargeBlueCoral', 'HappyCyanRaft', 'SwimmingLightSeaGreenSeal'
+      ].each_with_index do |username, i|
```

Comment:
```suggestion
      # Please update this spec if module constants are updated
      [
        'LargeBlueCoral', 'HappyCyanRaft', 'SwimmingLightSeaGreenSeal'
      ].each_with_index do |username, i|
```
We should add a comment here just to let folks know that it's expected this'll need updating if data is updated